finances/finance-2015-q3.md

```diff
@@ -43,5 +49,5 @@
 * kept safe by Hugo: 6688.65 - 300 = 6388.65
 * kept safe by Michiel: -1.67 - 300 - 18 + 300 = -19.67
 * kept safe by Jimm: -600
-* kept safe by Pierre: 0 - 60 = -60
+* kept safe by Pierre: 0 - 60 - 180 - 40 - 60 - 60 - 140 - 40
```

Comment:
Please also update the total at the end of the line, then the total of this section, and also the total at the start of Q4 in the -q4.md file.